_profiles/drupal/resource-versioning/index.md

```diff
@@ -0,0 +1,301 @@
+| `d`      | `g`              | `h`            | `a`                   | `e`                 | `b`, `c`             | `e`                       |
+| `e`      | `g`              | `h`            | `a`                   | `g`                 | `d`                  | `f`                       |
+| `f`      | `g`              | `h`            | `e`                   | `g`                 | `e`                  | `g`                       |
+| `g`      | no link          | `h`            | `e`                   | no link             | `f`                  | `h`                       |
+| `h`      | `g`              | no link        | `g`                   | no link             | `g`                  | no link                   |
+
+> Note: In this example, `f` has both a `predecessor-version` and
+> `prior-working-copy` link to `e` because `e` was a version and `e` was also the
+> revision to which changes could be applied prior to `f`'s creation.
+
+# Version History
+
+A server **MAY** provide a "version history" endpoint. The primary data
+of a response document from a version history endpoint must be a collection of
+resource objects.
+
+Unless an `id` contains version information, the `type` and `id` members of each
+resource object in the collection **MUST** be the same.
+
+If provided, resource objects' `self` links **MUST NOT** be the same.
+
+# Version Negotiators
+
+This profile defines a number of version negotiators which a server **MAY**
+implement. Additional version negotiator's may be added to this profile at a
+later date.
+
+Version negotiators which are not defined by this profile **MUST** adhere to the
+same constraints as [implementation-specific query parameter names](https://jsonapi.org/format/1.1/#query-parameters-custom).
+
+It is **RECOMMENDED** that any alternative version negotiators be added to this
+profile. New version negotiators may be registered by sending one, joint email
+to the profile editors with the subject line: "New JSON:API Resource Version
+Negotiator: {negotiator name}". This will begin a process of refinement and/or
+result in a determination of fitness for addition to this profile.
+
+## ID-Based Version Negotiator
+
+This profile establishes the `id` version negotiator. An `id`-based version
+identifier is composed of two segments—the `id` version negotiator and a single
+version argument. Any colons (U+003A COLON, “:”) present in the version
+identifier after the first occurrence **MUST** be interpreted as part of the
+single version argument and **MUST NOT** be interpreted as a segment delimiter.
+
+The resource version returned for any given version argument in an `id`-based
+version identifier **MUST NOT** change over time.
+
+> Note: This profile is agnostic about the format of the version argument in
+> `id`-based version identifiers. For example, one server may use integers as
+> revision IDs, another may use UUIDs and yet another may use content-based
```

Comment:
This note could go away by defining it as an opaque string, like I suggested above.